### ISSUE TITLE
Problem with 'undefind is not a function'

### ISSUE THREAD

**Issue by GeWa** [parsnick/laravel-elixir-bowerbundle#4]
The problem occured by running gulp

```
[10:50:49] TypeError: undefined is not a function
    at bundleCss (project\node_modules\laravel-elixir-bowerbundle\index.js:119:41)
    at null.definition (project\node_modules\laravel-elixir-bowerbundle\index.js:65:45)
    at Task.run (project\node_modules\laravel-elixir\Task.js:94:17)
    at Gulp.<anonymous> (project\node_modules\laravel-elixir\index.js:89:52)
    at module.exports (project\node_modules\gulp\node_modules\orchestrator\lib\runTask.js:34:7)
    at Gulp.Orchestrator._runTask (project\node_modules\gulp\node_modules\orchestrator\index.js:273:3)
    at Gulp.Orchestrator._runStep (project\node_modules\gulp\node_modules\orchestrator\index.js:214:10)
    at Gulp.Orchestrator.start (project\node_modules\gulp\node_modules\orchestrator\index.js:134:8)
    at runNextSet (project\node_modules\laravel-elixir\node_modules\run-sequence\index.js:88:16)
    at Gulp.onTaskEnd (project\node_modules\laravel-elixir\node_modules\run-sequence\index.js:77:5)
[10:50:49] Error in plugin 'run-sequence'
Message:
    An error occured in task 'bower'.
```

On the line is:
`.pipe($.if(config.production, $.minifyCss()))`

We replaced it with:
`.pipe($.if(config.production, $.cssnano(config.css.cssnano.pluginOptions)))`

**Comment by robvanbentem** [comment]
I experienced the same issue, although my error was a bit more descriptive:

```
[18:52:26] 'bower' errored after 223 ms
[18:52:26] TypeError: $.minifyCss is not a function
    at bundleCss (C:\dev\projects\vdm-mostmail\node_modules\laravel-elixir-bowerbundle\index.js:118:41)
    at null.definition (C:\dev\projects\vdm-mostmail\node_modules\laravel-elixir-bowerbundle\index.js:64:45)
    at Task.run (C:\dev\projects\vdm-mostmail\node_modules\laravel-elixir\Task.js:94:17)
    at Gulp.<anonymous> (C:\dev\projects\vdm-mostmail\node_modules\laravel-elixir\index.js:89:52)
    at module.exports (C:\dev\projects\vdm-mostmail\node_modules\orchestrator\lib\runTask.js:34:7)
    at Gulp.Orchestrator._runTask (C:\dev\projects\vdm-mostmail\node_modules\orchestrator\index.js:273:3)
    at Gulp.Orchestrator._runStep (C:\dev\projects\vdm-mostmail\node_modules\orchestrator\index.js:214:10)
    at Gulp.Orchestrator.start (C:\dev\projects\vdm-mostmail\node_modules\orchestrator\index.js:134:8)
    at runNextSet (C:\dev\projects\vdm-mostmail\node_modules\run-sequence\index.js:88:16)
    at runSequence (C:\dev\projects\vdm-mostmail\node_modules\run-sequence\index.js:99:2)
[18:52:26] Error in plugin 'run-sequence'
Message:
    An error occured in task 'bower'.
```

This happened with version 2.1.1 at line 118

The fix @GeWa posted solves it for me as well, thanks!

**Comment by parsnick** [comment]
Thanks both, just pushed the fix in v2.1.2
